plugins/catalog-backend/src/modules/core/UrlReaderProcessor.ts

```diff
@@ -83,6 +83,14 @@ export class UrlReaderProcessor implements CatalogProcessor {
         })) {
           parseResults.push(parseResult);
           emit(parseResult);
+          if (parseResult.type === 'entity') {
+            emit(
+              processingResult.refresh(
+                stringifyEntityRef(parseResult.entity),
+                item.url,
```

Comment:
One thing we realised reviewing is that glob pattern locations won't support immediate refreshes of new files that are added since the glob pattern is expanded only to files that already exist. Do you have any ideas on how we could express glob patterns as refresh keys?

We where thinking that we might end up with the possibility of emitting a pattern like `https://example.com/foo/*` and matching that either in the database or a separate processor method, but we're open to ideas.

Probably let's skip it for this PR and leave it as an unknown problem for the future.